frontend/src/styles/AppFooter.css

```diff
@@ -1,3 +1,16 @@
 .AppFooter {
     /* Styles go here */
-}
\ No newline at end of file
+    background-color: #7FC1AC;
+    position: absolute;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
```

Comment:
Let's add layout css into as low level as possible, i.e.
- positioning (position, width) in the ...Page components
- container flow (display, justify, align) in main Footer component

Reply:
Will be resolved with a bugfix in a later sprint